ote_cli/ote_cli/tools/train.py

```diff
@@ -134,8 +146,14 @@ def main():
             },
         )
 
+    if args.enable_hpo:
+        lr_finder_enabled = run_hpo(args, environment, task_class, dataset)
```

Comment:
What if you will use `template.task_type` instead of `task_class` here? implementation will become more readable.

Put `if task_type == TaskType.CLASSIFICATION` there https://github.com/openvinotoolkit/training_extensions/pull/799/files#diff-e7d551945b43d89428b358e2eab64caea887f57a3630abe2ac31f91ea81f6a35R49

Reply:
and there https://github.com/openvinotoolkit/training_extensions/pull/799/files#diff-e7d551945b43d89428b358e2eab64caea887f57a3630abe2ac31f91ea81f6a35R63

Reply:
I replaced task_class with task_type. Thank you!